Camera.js

```diff
@@ -47,7 +47,7 @@ export default class Camera extends Component {
     FlashMode: CameraManager.FlashMode,
     TorchMode: CameraManager.TorchMode
   };
-  
+
```

Comment:
While I'm bugging you, revert these changes to the whitespace before you submit the PR to his repo. I believe you didn't do this, maybe Atom changed them somehow. But these extra characters are his coding style preference, and changing them just adds noise to the PR.